.github/scripts/release-plugins.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+git config --global user.email "githubbot@gluonhq.com"
+git config --global user.name "Gluon Bot"
+
+CLIENT_MAVEN_REPO_SLUG=abhinayagarwal/client-maven-plugin
+CLIENT_GRADLE_REPO_SLUG=abhinayagarwal/client-gradle-plugin
+
+cd $HOME
+git clone --depth 5 https://github.com/$CLIENT_MAVEN_REPO_SLUG
+git clone --depth 5 https://github.com/$CLIENT_GRADLE_REPO_SLUG
+
+###############################
+# 
+# Release client-maven-plugin
+#
+###############################
+cd $HOME/client-maven-plugin
+# Update Substrate
+mvn versions:set-property -Dproperty=substrate.version -DnewVersion=$1 -DgenerateBackupPoms=false
+git commit pom.xml -m "Update Substrate version to $1"
+# Remove SNAPSHOT
+mvn versions:set -DremoveSnapshot
+CLIENT_PROJECT_VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
+# Commit and push tag
+git commit pom.xml -m "Release $CLIENT_PROJECT_VERSION"
+git tag $CLIENT_PROJECT_VERSION
+git push https://abhinayagarwal:$2@github.com/$CLIENT_MAVEN_REPO_SLUG $CLIENT_PROJECT_VERSION
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=5592359)